examples/iot/index.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+import * as iot from "@pulumi/azure/iot";
+
+const location = "West US 2";
+
+const resourceGroup = new azure.core.ResourceGroup("test", {
+    location: location,
+});
+
+const iotHub = new iot.IoTHub("test", {
+    resourceGroupName: resourceGroup.name,
+    sku: {
+        capacity: 1,
+        name: "S1",
+        tier: "Standard",
+    },
+    fallbackRoute: {
+        source: "DeviceMessages", 
+        enabled: true, 
+        endpointNames: ["events"],
+        condition: "true"
+    }
```

Comment:
Super Nit: missing comma on two lines above

Reply:
Sorry - I don't see what you mean. There are commas in lines 21 and 22.

Reply:
Line 23 and 24 - it's common to add trailing commas to every line of a multi-line object literal
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
fixed in 322489e874aff909e0d04b962f58eeada1412ea5